network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -404,3 +405,45 @@ class GeneralProductPage(ProductPage):
 
     class Meta:
         verbose_name = "General Product Page"
+
+
+class BuyersGuidePage(FoundationMetadataPageMixin, Page):
+    max_count = 1
```

Comment:
let's not preemptively restrict the use.
```suggestion
```

Reply:
This is going to be the new /privacynotincluded/ page which comes with a lot of automation; are we sure we want to allow more than one of these to exist? @Pomax

Reply:
Im going to remove this now since its fast enough for us to add back in should we need to. Marking as resolved.